docs/getting_started.rst

```diff
@@ -152,6 +152,7 @@ the index '0' corresponds to the first chain in the ``StanFit``:
 .. code-block:: python
 
     chain_1 = bern_fit.sample[:,0,:]
+    chain_1
```

Comment:
maybe you could add example what is seen with this or maybe use `.shape` (I don't remember what this should show)

    chain_1.shape  # (1000,)

Reply:
while this illustrates the structure of the numpy.ndarray, it doesn't fit will with the downstream analysis workflow.  a better example would be showing how to extract a single draw - either at random (random draw, random chain, all columns) - or the 1st draw from the 1st chain.

Reply:
I just put in both as an example - I thought a line more shouldn't hurt and in my experience the more detailed the tutorial the better.